bahmnicore-api/src/test/java/org/bahmni/module/bahmnicore/forms2/util/FormUtilTest.java

```diff
@@ -100,23 +100,23 @@ public void shouldReturnObsWhichHaveFormFieldPath() {
     }
 
     @Test
-    public void shouldReturnFormNameAlongWithVersionForGivenFormFieldPath() {
+    public void shouldReturnParentFormFieldPathForGivenFormFieldPath() {
```

Comment:
We do not have an actual test for lastIndexOf, it will still pass all tests if indexOf is called. Can we add the required tests?

Reply:
Added.